app/controllers/toggle-3ds/3ds.get.controller.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+// NPM dependencies
+const lodash = require('lodash')
+
+// Local dependencies
+const { response, renderErrorView } = require('../../utils/response')
+const { ConnectorClient } = require('../../services/clients/connector_client')
+const { correlationHeader } = require('../../utils/correlation_header')
+
+module.exports = async (req, res) => {
+  const connector = new ConnectorClient(process.env.CONNECTOR_URL)
+  const correlationId = req.headers[correlationHeader] || ''
+  const accountId = req.account.gateway_account_id
+
+  try {
+    const { card_types: acceptedCards } = await connector.getAcceptedCardsForAccountPromise(accountId, correlationId)
+
+    const pageData = {
+      disabled3ds: req.account.disableToggle3ds,
+      supports3ds: req.account.supports3ds,
+      requires3ds: req.account.requires3ds,
+      hasCardTypeRequiring3dsEnabled: lodash.some(acceptedCards, 'requires3ds'),
+      showHelper3ds: req.account.payment_provider === 'worldpay'
+    }
+
+    return response(req, res, 'toggle-3ds/index', pageData)
+  } catch (error) {
+    return renderErrorView(req, res, error.message.message[0], error.errorCode)
```

Comment:
I think the error message passed in here is displayed to the user. We probably don't want to display an error that comes from connector.